docs(tutorials): review pt 3 MTA-1685c (#424)
scaleway/docs-content: tutorials/jitsi-debian/index.mdx

  <Message type="tip">

>     In case the generation of the Let's Encrypt certificate fails, run the following command: `dpkg-reconfigure jitsi-meet-web-config` to reconfigure the package `jitsi-meet-web-config` before relaunching the certificate request.

Reviewer: ```suggestion
    If the generation of the Let's Encrypt certificate fails, run the following command: `dpkg-reconfigure jitsi-meet-web-config` to reconfigure the package `jitsi-meet-web-config` before relaunching the certificate request.
```
